AggregateMetrics/AggregateMetrics/Two/PercentileAggregations.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.Two
```

Comment:
nitpick: StyleCop compliance, particularly doc comments, would be great for all new code.